docs/DeveloperGuide.md

```diff
@@ -357,14 +358,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User requests to find TAs containing a specific keyword
-2.  TAlent Assistant™ returns list of TAs found with given keyword in specified attribute field
+1. User requests to find TAs containing specific keyword(s)
+2. TAlent Assistant™ returns a list of TAs that contain one or more of the given keyword(s) 
+in the specified attribute field
 
     Use case ends.
 
 **Extensions**
 
-* 2a. No TA applicant entries contain the given keyword
+* 2a. No TA candidate entries contains the given keyword in the specified attribute field
```

Comment:
Sorry something super minor here, but should the word contain be singular instead? (I might be wrong my proficiency in English is 👎🏻 )

Reply:
No no, thanks for spotting I think you're right!! :) I'll make the quick change!

Reply:
I am confused 😖